tools/beman-tidy/beman_tidy/lib/checks/beman_standard/repository.py

```diff
@@ -71,4 +74,41 @@ def fix(self):
         )
 
 
-# TODO REPOSITORY.DISALLOW_GIT_SUBMODULES
+@register_beman_standard_check("REPOSITORY.DISALLOW_GIT_SUBMODULES")
+class RepositoryDisallowGitSubmodulesCheck(FileBaseCheck):
+    def __init__(self, repo_info, beman_standard_check_config):
+        super().__init__(repo_info, beman_standard_check_config, ".gitmodules")
+
+    def pre_check(self):
+        # Need to override this, because REPOSITORY.DISALLOW_GIT_SUBMODULES is conditional
+        return True
+
+    def check(self):
+        if self.path.exists():
+            content = self.read()
+
+            # Regex pattern to match "wg21" submodule
+            regex = re.compile(
+                textwrap.dedent(r"""
+                    ^\[submodule \"(?:.+?/)?wg21\"]
+                    \tpath = (.+?)
```

Comment:
```suggestion
                    \tpath = papers/(.+?)
```
actually the path should start with papers, according to DIRECTORY.PAPERS. we should also incorporate this observation here.